### PR TITLE
Check for empty title

### LINKED ISSUE #11: Validate post title to ensure it's not empty
And maybe content too?

### PR DESCRIPTION
**Pull request by pranavmangal**
Closes #11 

Also fixes the summary length check

### CODE REVIEW

**Comment by hasali19** [comment]
Shouldn't a post always have content?

**Comment by pranavmangal** [comment]
> Shouldn't a post always have content?

I thought it would be good to have the ability to post just a file (or files) with a title. I feel for some cases, a title might be descriptive enough to not require additional content.

**Comment by hasali19** [comment]
Sorry, maybe we should have merged this earlier. Could you fix the conflicts.

**Comment by hasali19** [comment]
@adamdejl 
Can this be merged? I think it's removing some of the extra checks you added.

File: drp/api/posts.py [hunk 1]
```diff
@@ -173,18 +173,16 @@ def post(self):
         files = request.files.getlist('files')
         names = request.form.getlist('names')
 
-        if title is None or summary is None or content is None:
```
Comment (adamdejl) [inline drp/api/posts.py:176]: I would perhaps suggest recovering these checks before merging. They are probably not vital, but I still think they are useful as a sanity check. As the title, summary and content fields are always visible on the frontend, it wouldn't probably make much sense to send a request without one of them.

Comment (hasali19) [reply drp/api/posts.py:176]: Yeah I think we want to make it more restrictive, not less. Ideally we should check that they are all not `None` and also not empty.

Comment (pranavmangal) [reply drp/api/posts.py:176]: Alright, I will add them back in

**Pull request merged**